Drop-In/src/main/java/com/braintreepayments/api/DropInRequest.java

```diff
@@ -98,9 +94,9 @@ public DropInRequest collectDeviceData(boolean collectDeviceData) {
     /**
      * This method is optional.
      *
-     * @param request The Google Payment Request {@link GooglePaymentRequest} for the transaction.
+     * @param request The Google Payment Request {@link GooglePayRequest} for the transaction.
      */
-    public DropInRequest googlePaymentRequest(GooglePaymentRequest request) {
+    public DropInRequest googlePaymentRequest(GooglePayRequest request) {
```

Comment:
Is renaming this (and the setter) to use `googlePay` instead of `googlePayment` going to be part of a separate PR?

Reply:
Yes we will do this in a separate PR - added a TODO for it